action.yml

```diff
@@ -12,7 +12,7 @@ inputs:
     default: 'Reply to'
   tweetContent:
     description: 'Heading where the tweet content is stored'
-    default: 'Content'
+    default: 'Tweet Content'
```

Comment:
Happy to revert if we want to go with `Content`. However I find `Tweet Content` more descriptive.

Reply:
The default I had matched the example config of the monolithic cq which supported (optionally) sharing the content between twitter and mastodon etc. which is wjy it was just called Content by default. I think it was explicitly designed to allow splitting them, or even have separate sections for separate accounts iirc (which is obviously still easily possible with the actions setup)